benchmarks/DeepLearning/Layers/RMSNorm/GoogleBenchmarkMain.cpp

```diff
@@ -0,0 +1,147 @@
+//===- GoogleBenchmarkMain.cpp---------------------------------------------===//
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+//===----------------------------------------------------------------------===//
+//
+// This file implements the benchmark for RMSNORM layer.
+//
+//===----------------------------------------------------------------------===//
+
+#include <benchmark/benchmark.h>
+#include <buddy/Core/Container.h>
+#include <iostream>
+#include <random>
+
+// Define target layout.
+#define INPUT_DIM 256
+#define BATCH_SIZE 1
+#define OUTPUT_DIM 256
+constexpr size_t ParamsSize = 256;
+
+// Helper functions and variables.
+namespace {
+const std::string PASS = "\033[32mPASS\033[0m";
+const std::string FAIL = "\033[31mFAIL\033[0m";
+
+bool areArraysEqual(float array1[], float array2[], int size) {
+  for (int i = 0; i < size; ++i) {
+    if (array1[i] != array2[i]) {
```

Comment:
The result of this benchmark is "Pass," indicating that the calculation result is correct. However, directly using the equality operator (`==`) to compare floating-point numbers may not be the best practice. This is because floating-point calculations can introduce tiny precision errors. 
To more reliably compare floating-point numbers, it's advised to use a small error margin, for instance: [MobileNetV3 example](https://github.com/buddy-compiler/buddy-benchmark/blob/main/benchmarks/DeepLearning/Models/MobileNet-V3/GoogleBenchmarkMain.cpp#L44)